content/docs/command-reference/root.md

```diff
@@ -10,11 +10,11 @@ usage: dvc root [-h] [-q | -v]
 
 ## Description
 
-This command returns the path to the root directory of the <abbr>DVC
-project</abbr>, relative to the current working directory. It can be used to
-build a path to a dependency, script, or <abbr>data artifact</abbr>, for
-example. Useful when working in a subdirectory of the project, and needing to
-refer to a file in another directory.
+Returns the path to the root directory of the <abbr>DVC project</abbr>, relative
+to the current working directory. Useful when working in a subdirectory of the
+project, and you need to refer to a file in another directory. Use it in files
+and commands to build a path to a dependency, script, or <abbr>data
```

Comment:
> Use it in files and commands

How do you use a command in files and commands? I'm not sure what we mean here 🙂

Reply:
Yes, this part is confusing, probably because there are effectively two usages: `dvc root` the command and `$(dvc root)` used in another command or script. I think this will make more sense and can be rewritten once clarify the recommended usage.

Reply:
And probably worth it's own paragraph after all. I did come to conclusions in https://github.com/iterative/dvc.org/issues/404#issuecomment-666104957 but feel free to wait a bit and see if the team has more feedback. Thanks!

Reply:
Split to second paragraph and adjusted language — instead of "files and commands" the primary motivation is "build reusable paths" as per discussion on #404.

Reply:
OK, better, thanks. Resolving here but I have more feedback on this...